composer.json

```diff
@@ -484,6 +484,10 @@
             "@composer scenario phpredis4",
             "@composer test -- tests/Integrations/PHPRedis/PHPRedis4SandboxedTest.php"
         ],
+        "test-phpredis5": [
```

Comment:
These aren't running in CI since they haven't been added to `test-integrations-7*`.

Reply:
good catch Sammy